safenode/src/bin/kadclient.rs

```diff
@@ -101,11 +98,11 @@ async fn main() -> Result<()> {
         }
 
         for xorname in chunks_to_fetch.iter() {
-            println!("Fetching chunk {xorname:?}");
-            match client.get_chunk(ChunkAddress::new(*xorname)).await {
-                Ok(chunk) => info!("Successfully got chunk {}!", chunk.name()),
+            println!("Downloading file {xorname:?}");
```

Comment:
this is a chunk downloaded, not the whole file

Reply:
No, @maqi , check the innards of `read_bytes`. It gives a whole file.

It takes a chunk name, checks if it is a head chunk asked for (the one holding the data map, or the final one if data map chunkling resulted in more than one chunk).